internal/cli/get/get.go

```diff
@@ -104,6 +107,49 @@ kargo get stages my-project -o json
 	return cmd
 }
 
+func printStageResult(opt *option.Option, res runtime.Object) error {
+	if pointer.StringDeref(opt.PrintFlags.OutputFormat, "") != "" {
+		return printResult(opt, res)
+	}
+	var items []runtime.RawExtension
+	if list, ok := res.(*metav1.List); ok {
+		items = list.Items
+	} else {
+		items = []runtime.RawExtension{{Object: res}}
+	}
+	table := &metav1.Table{
+		ColumnDefinitions: []metav1.TableColumnDefinition{
+			{Name: "Name", Type: "string"},
+			{Name: "Current Freight", Type: "string"},
+			{Name: "Health", Type: "string"},
+			{Name: "Age", Type: "string"},
+		},
+		Rows: make([]metav1.TableRow, len(items)),
+	}
+	for i, item := range items {
+		// This func is only ever passed Stages
```

Comment:
Is this enforced somehow? If not, should we duplicate this comment at the start of the function?

Reply:
Technically, nothing enforces it, but the function name should hopefully be a dead giveaway.